.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: release
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      tag:
+        description: 'tag and release to create'
+        # Input has to be provided for the workflow to run
+        required: true
+env:
+  GITHUB_ENV: ".env"
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - name: run make bump-version
+        run: make bump-version
+        env:
+          NEW_VERSION: ${{ github.event.inputs.tag }}
+      - name: commit manifests
+        uses: EndBug/add-and-commit@a988073222b8bd50f3ecfca9c0ab7dfbf0d08ceb
+        with:
+          message: "create release manifests for ${{ github.event.inputs.tag }}"
+          add: "releases/"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # target commit should be the commit with the new release manifest, not the commit when workflow was triggered
+      - name: find target commit to tag
+        run: |
+          sha=$(git rev-parse HEAD)
+          echo "TARGET_SHA=$sha" >> $GITHUB_ENV
+      - name: Create GitHub Release
+        uses: actions/create-release@1e92f6fc665e271a5435631ba00711fdd83d9d27
```

Comment:
Is this the same Action referenced by https://github.com/actions/create-release? That repo is archived, is that a reason to be concerned?

Reply:
Ah, okay. Didn't verify that. Will switch to https://github.com/softprops/action-gh-release